pkg/controllers/job/helpers/helpers.go

```diff
@@ -35,7 +36,7 @@ const (
 )
 
 // GetTaskIndex returns task Index.
-func GetTaskIndex(pod *v1.Pod) string {
+func GetPodIndex(pod *v1.Pod) string {
```

Comment:
Let's use `Task` and `TaskInstance` :)

Reply:
@Thor-wl 

I think the function of this function is to find the index of the pod in the task, so maybe the name `GetPodIndex` is more appropriate?

Reply:
Indeed, it's an awkward point that `Task` in the code represents `Pod` most through the project. It is easily confused for new comers to make difference.
Do you have some good ideas?
@william-wang @shinytang6 @jasonliu747

Reply:
Seems "GetTaskIndex" is the best for two places:)  Let's use GetTask(pod *v1.Pod) here and use the GetTaskIndex(taskName, job) for your new code change.

Reply:
@william-wang ok, that loosk good.